docs/fast_data/connectors/debezium_cdc.md

```diff
@@ -526,6 +526,240 @@ debezium.transforms.Reroute.key.enforce.uniqueness=false
 </details>
 
 
+### DB2
+
```

Comment:
I was thinking, since we tested that only with a DB2 Docker container, should we add that note? Should we also provide a link to the official IBM documentation, so that developers can check it out in case they would like to get deeper insights on how the database works?

What do you think?

Reply:
I agree

Reply:
Changed (I wrote this some lines below)